app/assets/javascripts/sprangular/services/account.coffee

```diff
@@ -52,9 +52,11 @@ Sprangular.service "Account", ($http, _, $q, Cart, Flash) ->
       $http.post '/spree/login.json', $.param params
         .success (data) ->
           service.populateAccount(data)
-          Flash.success 'Successfully signed in'
+          $translate('app.signed_in').then (paragraph) ->
+            Flash.success paragraph
         .error ->
-          Flash.error 'Sign in failed'
+          $translate('app.signin_failed').then (paragraph) ->
```

Comment:
Agreed,  to be fair. I haven't been thinking much on this.  Just bulldozing.  Give me a sec and i'll refactor Flash.